src/index.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
+
+import * as v1 from './v1';
+import * as v1beta1 from './v1beta1';
+import * as v1p1beta1 from './v1p1beta1';
+import * as v1p2beta1 from './v1p2beta1';
+const AssetServiceClient = v1.AssetServiceClient;
+export {v1, v1beta1, v1p1beta1, v1p2beta1, AssetServiceClient};
+// For compatibility with JavaScript libraries we need to provide this default export:
+// tslint:disable-next-line no-default-export
+export default {v1, AssetServiceClient};
```

Comment:
we should also list other versions in the default export, otherwise it will be a breaking change:

```
> require('@google-cloud/asset')
{
  AssetServiceClient: [Function: AssetServiceClient],
  v1p2beta1: { AssetServiceClient: [Function: AssetServiceClient] },
  v1beta1: { AssetServiceClient: [Function: AssetServiceClient] },
  v1: [Circular],
  default: {
    AssetServiceClient: [Function: AssetServiceClient],
    v1p2beta1: { AssetServiceClient: [Function: AssetServiceClient] },
    v1beta1: { AssetServiceClient: [Function: AssetServiceClient] },
    v1: [Circular]
  }
}
```